lego/apps/emojis/models.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.contrib.postgres import fields
+from django.db import IntegrityError, models
+from rest_framework.exceptions import ValidationError
+
+from lego.apps.emojis.permissions import EmojiPermissionHandler
+from lego.utils.managers import BasisModelManager
+from lego.utils.models import BasisModel
+
+
+class Emoji(BasisModel):
+    id = models.CharField(max_length=40, primary_key=True)
+    keywords = fields.ArrayField(models.CharField(max_length=40))
+    unicode_string = models.CharField(max_length=24, db_index=True)
+    fitzpatrick_scale = models.BooleanField()
```

Comment:
How do we handle reactions with emojis that can have different skin colors?

Reply:
That's what this field is for, however it isn't used now. I'd rather focus on it later on, might require some unique constrains if we decide on using skin colors.